pkg/operator/starter.go

```diff
@@ -281,6 +287,7 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 		operatorCtx.kubeInformersForNamespaces,
 		operatorCtx.operatorConfigInformer,
 		operatorCtx.resourceSyncController,
+		clusterVersionConfig.Status.Capabilities.EnabledCapabilities,
```

Comment:
turn it into a set and distribute that to avoid needless for loops when trying to figure out which capability is enabled